### DISCUSSION TITLE
求助！怎么编译 release 包

### DISCUSSION THREAD

**Discussion by Aarontaken** [fcitx5-android/fcitx5-android#627] [Q&A | 问答] [answered]
我想自己编译一个版本需要怎么配置，现在debug版本能编译安装，但是编release包一堆c相关的报错。有大佬可以解答吗，本意是想改改默认键盘高度主题啥的。。
![image](https://github.com/user-attachments/assets/0f7b3f50-cfcc-4e3d-adb2-d4c352a09772)
![image](https://github.com/user-attachments/assets/798695ae-d62e-4a59-b823-84f68b1d93bb)

**Comment by rocka** [comment, marked as answer]
Android Studio 内双击 <kbd>Shift</kbd> 呼出搜索框，查找 "Generate Signed App Bundle / APK"

<img src="https://github.com/user-attachments/assets/bcb29667-68be-49ed-b10b-caa007eb8f80" width="407">


或者命令行运行 `./gradlew app:assembleRelease`
